src/commands/increment-version.ts

```diff
@@ -22,7 +22,7 @@ import { getChangelogText } from './create-changelog';
 const BADGE = '[increment-version]\t';
 
 /**
- * Increments the pre-version in `package.json` automatically, leave the "base"-version alone.
+ * Increments the pre-version in `package.json` automatically, leaving the "base" of the version alone.
```

Comment:
```suggestion
 * Increments the pre-version in `package.json` automatically.
```

Ich denke das nachfolgende Beispiel illustriert das schon ganz gut. 
Außerdem wird das weiter unten nochmal erwähnt.